gpkit/__init__.py

```diff
@@ -92,7 +92,7 @@ def __enter__(self):
         global SIGNOMIALS_ENABLED
         SIGNOMIALS_ENABLED = True
 
-    def __exit__(self, type, value, traceback):
+    def __exit__(self, type_, val, traceback):
```

Comment:
(this was to get pylint to stop complaining about overwriting `value` from the outer scope)